<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketEditForm</class>
 <widget class="QWidget" name="SocketEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>130</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="Title2Label" name="socket_title_label">
     <property name="text">
      <string>Socket X</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BoldLabel" name="socket_name_label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="socket_name_line_edit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BoldLabel" name="socket_description_label">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="socket_description_line_edit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BoldLabel</class>
   <extends>QLabel</extends>
   <header>BoldLabel.h</header>
  </customwidget>
  <customwidget>
   <class>Title2Label</class>
   <extends>QLabel</extends>
   <header>Title2Label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
